dome/install_osx.md

```diff
@@ -19,7 +19,7 @@ brew install ignition-dome
 
 All libraries should be ready to use and the `ign gazebo` app ready to be executed.
 
-Head back to the [Getting started](/docs/get_started)
+Head back to the [Getting started](/get_started)
```

Comment:
I think the correct link is `/docs/all/get_started`

Reply:
5a19708659f5f91470480ad9f8dfc9238c2ba85c